reflred/angles.py

```diff
@@ -37,6 +37,8 @@ def apply_divergence_simple(data, sample_width):
     dtheta = divergence_simple(slits=slits, distance=distance, T=theta,
                                sample_width=sample_width, use_sample=use_sample)
     #print("divergence", theta.shape, dtheta.shape)
+    if data.source_divergence is not None:
+        dtheta = np.minimum(dtheta, FWHM2sigma(data.source_divergence/2))
```

Comment:
I recommend converting source divergence to 1-σ on load rather than on use.  This is what is done elsewhere, such as [monochromator divergence](https://github.com/reductus/reductus/blob/main/reflred/candor.py#L150) in `candor.py`.

Reply:
I don't know the shape of the angle distribution coming out of the mirrors.  If it is gaussian with a sigma width, then we should use that.  If it's more trapezoidal or rectangular then I'm not sure 1-sigma makes sense, does it?

Reply:
Good point. However I still prefer 1-σ because inconsistency between 1-σ and FWHM has led to bugs in the past. 

Does it make sense to move all divergence to FWHM? How about wavelength dispersion?

1-σ is just the width of the second central moment and it is easy to recover the distribution width from it if you know the distribution is uniform. This is no different than recovering the Gaussian width from FWHM, which you would need to do if the distribution were normal instead of uniform.

I've been making the assumption that the divergence is uniform in π coming out of the beam pipe, though clearly it is narrower than that. Are there cases where the beam divergence is narrower than the slits would suggest for neutron sources? In which case we want to add this divergence limiter for neutron instruments as well.